samples/csharp_dotnetcore/24.bot-authentication-msgraph/README.md

```diff
@@ -20,10 +20,14 @@ NOTE: Microsoft Teams currently differs slightly in the way auth is integrated w
   dotnet --version
   ```
 
-- Update `appsettings.json` with required configuration settings
-  - MicrosoftAppId
-  - MicrosoftAppPassword
-  - ConnectionName
+- Update `appsettings.json` with required configuration settings 
+    | Property                  | Value Description     | 
+    |----------------------|-----------|
+    | MicrosoftAppType       | UserAssignedMSI  |
```

Comment:
I think MicrosoftAppType and MicrosoftAppTenantId aren't required fields.

Reply:
thanks! updated.